_posts/2018-12-19-bazel-0.21.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: posts
+title: Bazel 0.21
+authors:
+  - laurentlb
+---
+
+We've just released [Bazel 0.21](https://github.com/bazelbuild/bazel/releases/tag/0.21.0)!
+
+We also revamped our Android with Bazel tutorial. Check it out [here](https://docs.bazel.build/versions/master/tutorial/android-app.html).
+
+
+## Backward compatibility
+
+Before updating Bazel, you may try if anything would break with 0.21. To do so, build your code with Bazel 0.20 and the following flags:
+
+*   [`--incompatible_disable_sysroot_from_configuration`](https://github.com/bazelbuild/bazel/issues/6565)
+*   [`--incompatible_disallow_slash_operator`](https://github.com/bazelbuild/bazel/issues/5823)
+*   [`--incompatible_static_name_resolution`](https://github.com/bazelbuild/bazel/issues/5637)
+*   [`--incompatible_package_name_is_a_function`](https://github.com/bazelbuild/bazel/issues/5827)
+*   [`--incompatible_use_remotejdk_as_host_javabase`](https://github.com/bazelbuild/bazel/issues/6656)
+*   [`--incompatible_provide_cc_toolchain_info_from_cc_toolchain_suite`](https://github.com/bazelbuild/bazel/issues/6537)
+*   [`--incompatible_disable_depset_in_cc_user_flags`](https://github.com/bazelbuild/bazel/issues/6383)
+*   [`--incompatible_disable_late_bound_option_defaults`](https://github.com/bazelbuild/bazel/issues/6384)
+*   [`--experimental_strict_action_env`](https://github.com/bazelbuild/bazel/issues/6648)
+
+For more information about the changes, please click on the links and read the GitHub issues. Comment there if you need help.
+
+If you use external repositories, you might first have to update the dependencies in your WORKSPACE file.
+
+Changes for which 0.21 is a [migration window](
+https://docs.google.com/document/d/1Dj5PBLmPVg9ZyApm4GobM3y-mDgY3mVaqpRVttOe-ZQ/edit)
+are marked with GitHub label [migration-0.21](
+https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Amigration-0.21).
+
+Breaking changes in the next release (0.22) are marked with GitHub label
+[breaking-change-0.22](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Abreaking-change-0.22).
+
+
+## Android
+
+*   [`--experimental_check_desugar_deps`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--experimental_check_desugar_deps)
```

Comment:
All bazel.build links should point to the right version, not master.

Reply:
It's not available.
https://docs.bazel.build/versions/0.21.0/ doesn't exist yet.
We can improve our release workflow for 0.22